scripts/vote.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/zsh
+
+echo "vote for $1"
```

Comment:
Is this a seperate requirement ?

Reply:
It is part of the overall system used to test the fixes. Nor necessarily part of sifnoded on DevNet, TestNet, MainNet however locally used to see things working.